tools/ARIAtools/extractProduct.py

```diff
@@ -417,7 +416,7 @@ def finalize_metadata(outname, bbox_bounds, prods_TOTbbox, dem, lat, lon, mask=N
                 out_interpolated[iz, iline, ipix] = interp_2d(line, pixel, hgt)
     out_interpolated=np.flip(out_interpolated, axis=0)
     # must use 'nearest' interpolation to avoid disprepancies between different crop extents
```

Comment:
update this comment as this is no longer accurate.

Reply:
Ok, I fixed that.